deepchem/models/torch_models/gnn3d.py

```diff
@@ -559,27 +585,43 @@ def loss_func(self, inputs, labels, weights):
         torch.Tensor
             The computed loss value.
         """
-        # encodings2d = []
-        # encodings3d = []
-        # for conformers in inputs:
-        #     # 2d model takes only the first conformer
-        #     encodings2d.append(self.components['2d'](conformers[0]))
-        #     # 3d model takes all conformers
-        #     encodings3d.append(
-        #         [self.components['3d'](conf) for conf in conformers])
-
-        # # concat the lists such that the 2d encodings is of shape batch_size x target_dim
-        # # and the 3d encodings is of shape batch_size*num_conformers x target_dim
-        # encodings2d = torch.cat(encodings2d, dim=0)
-        # encodings3d = torch.cat(
-        #     [torch.cat(conf, dim=0) for conf in encodings3d], dim=0)
-        # NOTE: The above one is logically correct but the below method is similar
-        # to original implementation here: https://github.com/HannesStark/3DInfomax/blob/5cd32629c690e119bcae8726acedefdb0aa037fc/trainer/self_supervised_trainer.py#L26
-        # The different between above and below is that the below one
-        # cannot handle multiple conformers.
-        encodings2d = self.components['2d'](inputs)
-        encodings3d = self.components['3d'](inputs)
-        loss = self.criterion(encodings2d, encodings3d)
+        if self.task == 'pretraining':
+            # encodings2d = []
```

Comment:
Can we remove the commented out code or do we need it still?

Reply:
That was a leftover piece of code - I removed it as it is no longer required.